source/Nevermore/QueryBuilder.cs

```diff
@@ -230,8 +230,10 @@ public int Count()
         public bool Any()
         {
             var clonedSelectBuilder = selectBuilder.Clone();
-            clonedSelectBuilder.AddColumnSelection(new SelectExistsSource());
-            var result = transaction.ExecuteScalar<int>(new ExistsSelectBuilder(clonedSelectBuilder).GenerateSelect().GenerateSql(), paramValues);
+            clonedSelectBuilder.RemoveOrderBys();
+            clonedSelectBuilder.IgnoreDefaultOrderBy();
+            var query = new IfExpression(new ExistsExpression(clonedSelectBuilder.GenerateSelect()), new SelectConstant("1"), new SelectConstant("0")).GenerateSql();
```

Comment:
Eww, magic strings

Reply:
This code seems appropriate to me. What would you prefer to see instead?

Reply:
A `const` for true and false, which is then also used in the return below.

Reply:
Sure, thought you might have been thinking along different lines, that works :+1: